be/src/olap/delta_writer.cpp

```diff
@@ -113,13 +113,13 @@ OLAPStatus DeltaWriter::init() {
             .set_rowset_state(PREPARING)
             .set_txn_id(_req.txn_id)
             .set_load_id(_req.load_id);
-    RowsetBuilderContext builder_context = context_builder.build();
+    RowsetWriterContext builder_context = context_builder.build();
```

Comment:
if the class name is RowsetWriter then the param name should be writer_context?

Reply:
OK